modules/ocf_desktop/manifests/stats.pp

```diff
@@ -24,6 +24,16 @@
       mode   => '0555',
       owner  => ocfstats,
       source => 'puppet:///modules/ocf_desktop/stats/update-delay.sh';
+      
+    '/opt/stats/update-flags.sh':
+      mode   => '0555',
```

Comment:
`ibid` on permissions. especially here you use `who` instead of `whoami` in the which further hurts the AC